### ISSUE TITLE
Navigation bar is not working

### ISSUE THREAD

**Issue by ttSpace** [microsoft/PTVS#6551]
**Describe the bug**

![image](https://user-images.githubusercontent.com/82852072/120429146-7da74100-c3a7-11eb-8290-da65151e92f5.png)

**Steps to Reproduce**

![image](https://user-images.githubusercontent.com/82852072/120429788-95cb9000-c3a8-11eb-8812-0a912b253870.png)

**Expected behavior**

Navigate bar displayed at the top of each editor window and includes a tow-level list of definitions.		
i. Left drop-down contains top-level class and function definitions in the current file.
ii. Right drop-down displays a list of definitions within the scope shown in the lest.

**Comment by rchiodo** [comment]
This should be handled here:

https://dev.azure.com/devdiv/DevDiv/_git/VSLanguageServerClient?path=%2Fsrc%2Fproduct%2FRemoteLanguage%2FImpl%2FFeatures%2FNavbar%2FRemoteLanguageDropdownBarClient.cs&_a=contents&version=GBdevelop

Not sure why it isn't.

**Comment by rchiodo** [comment]
I don't get the nav bar. Not sure how this is showing up for you @ttSpace 

![image](https://user-images.githubusercontent.com/19672699/120507709-e8428780-c37b-11eb-9cce-b361b1870e12.png)

**Comment by rchiodo** [comment]
@ttSpace how did you get the nav bar to show up?

**Comment by ttSpace** [comment]
@rchiodo you can go to Tools > Options > Text Editor > Python > General > Navigation bar (The option is checked by default)
![image](https://user-images.githubusercontent.com/82852072/120570819-86515300-c44b-11eb-8d7d-94a1b8df7ac0.png)

**Comment by rchiodo** [comment]
Hmm, I have that set but I still don't see nav bars.

![image](https://user-images.githubusercontent.com/19672699/120701643-f241b480-c467-11eb-824e-6a02551b2888.png)

**Comment by greazer** [comment]
This should be fixed when goto definition gets fixed by #6541

**Comment by linette-zyy** [comment]
@greazer 
Go to definition works well on this build. But the Navigation bar is still not show up.
![image](https://user-images.githubusercontent.com/59192491/129828865-a7c68287-f3f8-4e8b-af9b-4c3eb05713ed.png)

**Comment by ttSpace** [comment]
In today's build I see the Navigation bar, but there is nothing in the the Navigation bar.

![image](https://user-images.githubusercontent.com/82852072/131941926-1b4b5b0c-4ea6-4c52-b73a-9794c4f8507e.png)

**Comment by ttSpace** [comment]
In toady's build ```17.1.0 Preview 3.0 [32030.14.main]``` , the Navigation bar still does not work properly.

**Comment by AdamYoblick** [comment]
This still needs to be investigated. If I had to guess, I'd say pylance is not properly sending the list of methods and properties to the LSP client or they are getting lost along the way.

**Comment by AdamYoblick** [comment]
This doesn't repro in vscode, so pylance is most likely doing the right thing here. PTVS or the VS LSP Client is most likely responsible.

**Comment by linette-zyy** [comment]
In today's latest build:**17.6.0 Preview 1.0 [33331.318.main]**,Python Package:**17.0.23031.1**
Navigation bar is missing and go to Tools > Options > Text Editor > Python > General > Navigation bar(it is disabled)
![image](https://user-images.githubusercontent.com/59192491/215964160-7d402b3e-add4-4ccc-a4a7-efb286cdb56f.png)

**Comment by StellaHuang95** [comment]
Yes that's disabled as a short term fix since the navigation bar isn't showing anything. Need to come up with a long term fix for this one.

**Comment by StellaHuang95** [comment]
Could you follow [this instruction](https://github.com/microsoft/PTVS/wiki/Build-and-Debug-Instructions-for-PTVS#:~:text=and%20try%20again.-,Install%20nodejs/npm,-from%20https%3A//nodejs) to install nodejs and try again?
